{
  "Header": {
    "Original": "CVE-2025-55551 (An issue in the component torch.linalg.lu of pytorch v2.8.0 allows att ...)",
    "ID": "CVE-2025-55551",
    "Description": "(An issue in the component torch.linalg.lu of pytorch v2.8.0 allows att ...)"
  },
  "Annotations": [
    {
      "Original": "- pytorch \u003cunfixed\u003e (bug #1116537)",
      "Type": "package",
      "Package": "pytorch",
      "Kind": "unfixed",
      "BugNo": 1116537
    },
    {
      "Original": "NOTE: https://github.com/pytorch/pytorch/issues/151401",
      "Type": "NOTE",
      "Description": "https://github.com/pytorch/pytorch/issues/151401"
    }
  ]
}